deploy.rb

```diff
@@ -11,6 +13,7 @@
 
 def update_repo(repo_dir)
   Dir.chdir(repo_dir) do
+    `git checkout config/deploy.rb`
```

Comment:
Can you say more about what this line is doing here in this PR?

Reply:
Not necessary for this PR. The purpose of this line is to remove the file change made on L81 so that the checkout of master can happen. This was a move from it's previous position on L69. This is a safer location, as it will always be run before the checkout of master.

This was provoked by a situation I encountered in testing in which the change to config/deploy.rb hadn't been removed and checkout of master failed.

Reply:
OK, sounds good. Thanks, @justinlittman